mods/default/locale/default.fr.tr

```diff
@@ -0,0 +1,211 @@
+# textdomain: default
+Locked Chest=Coffre verrouillé
+Locked Chest (owned by @1)=Coffre verrouillé (possédé par @1)
+You do not own this chest.=Ce coffre ne vous appartient pas.
+a locked chest=Un coffre verrouillé
```

Comment:
Is the uppercase "U" intended here? The original string starts with a lowercase letter.

Reply:
Indeed, it wasn't intended.